assets/css/page-widgets/get-pro.css

```diff
@@ -0,0 +1,20 @@
+.get-pro-link {
```

Comment:
This is basically a "normal" button right? Shouldn't the class name reflect that better?